cmd/althea/cmd/root.go

```diff
@@ -75,19 +158,23 @@ func initAppConfig() (string, interface{}) {
 	}
 
 	// DEFAULT SERVER CONFIGURATIONS
-	srvConfig := serverconfig.DefaultConfig()
```

Comment:
These configurations do all the flag/variable definitions for us without Canto's enforced defaults.